playbooks/operationalizing/secure-forward-splunk.adoc

```diff
@@ -0,0 +1,318 @@
+---
+---
+= Forwarding Logs to Splunk Using Fluentd Secure Forward
+Andrew Block <ablock@redhat.com>
+v1.0, 2017-02-05
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
```

Comment:
We should determine `scripts_repo` across all playbooks in general and update accordingly